gruntfile.js

```diff
@@ -1,6 +1,7 @@
 // Do not add new tasks. Grunt is used only for building and will be replaced.
 module.exports = function (grunt) {
   grunt.loadTasks('tasks')
+  grunt.util.linefeed = '\n'
```

Comment:
How does this line affect the build?

Reply:
In our case it shouldn't change anything, it's just telling grunt.util to use `LF` if needed. Being that grunt is going away from this repo, I thought we better be safe before that.